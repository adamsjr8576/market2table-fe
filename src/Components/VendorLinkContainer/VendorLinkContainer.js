import React, { useState } from 'react';
import './VendorLinkContainer.scss';
import VendorMarketList from '../../Containers/VendorMarketList/VendorMarketList';
import VendorMarketSearch from '../../Containers/VendorMarketSearch/VendorMarketSearch';
import PropTypes from 'prop-types';


export const VendorLinkContainer = () => {

  const [marketsLinked, setMarketsLinked] = useState([]);

  return (
    <section className='vendor-link-section'>
      <VendorMarketList marketsLinked={marketsLinked} setMarketsLinked={setMarketsLinked}/>
      <VendorMarketSearch marketsLinked={marketsLinked} setMarketsLinked={setMarketsLinked}/>
    </section>
  )
}

export default VendorLinkContainer;

VendorLinkContainer.propTypes = {
  marketsLinked: PropTypes.array,
  setMarketsLinked: PropTypes.func
}
