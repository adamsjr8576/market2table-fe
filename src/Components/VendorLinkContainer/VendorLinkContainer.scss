.vendor-link-section {
  height: 85vh;
  width: 100%;
  display: flex;
  align-items: center;
  justify-content: space-around;
}
