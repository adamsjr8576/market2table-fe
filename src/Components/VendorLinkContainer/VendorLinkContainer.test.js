import React from 'react';
import { shallow } from 'enzyme';
import VendorLinkContainer from './VendorLinkContainer';

describe('VendorLinkContainer', () => {

  let wrapper;

  it('should match the snapshot', () => {
    wrapper = shallow(<VendorLinkContainer />);

    expect(wrapper).toMatchSnapshot();
  });
});
