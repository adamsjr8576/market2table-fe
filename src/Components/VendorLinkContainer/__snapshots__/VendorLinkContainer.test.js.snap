// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VendorLinkContainer should match the snapshot 1`] = `
<section
  className="vendor-link-section"
>
  <VendorMarketList
    marketsLinked={Array []}
    setMarketsLinked={[Function]}
  />
  <VendorMarketSearch
    marketsLinked={Array []}
    setMarketsLinked={[Function]}
  />
</section>
`;
